app/controllers/concerns/exception_handler.rb

```diff
@@ -3,22 +3,11 @@ module ExceptionHandler
 
   included do
     rescue_from ActiveRecord::RecordNotFound do |e|
-      json_response({ message: e.message }, :not_found)
+      render json: { message: e.message }, status: :not_found
```

Comment:
I suggest the use of `rescue_from` at the `application_controller`, like:


```ruby
  rescue_from ActiveRecord::RecordNotFound, with: :resource_not_found

  def resource_not_found
    render json: { message: e.message }, status: :not_found
  end
```

Reply:
Changes integrated